Library/Formula/gd.rb

```diff
@@ -34,6 +34,14 @@ class Gd < Formula
     cause "Undefined symbols when linking"
   end
 
+  # 2.1.1: fix missing prefixes in webpimg.c
+  # https://github.com/Homebrew/homebrew/issues/49186
+  patch do
+    url "https://raw.githubusercontent.com/konqui/patches/master/gd/0001-fixed-a-few-typos-missing-VPX_-prefixes-in-webpimg.c.patch"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!